pkg/lib/editor/js/core-config-setup/config-setup-tool.html

```diff
@@ -745,6 +745,23 @@
                 </div>
               </td>
             </tr>
+            <tr>
+              <td>Sign JWTs: </td>
```

Comment:
Add brief description about what this is, why it is needed, and reminding users to add it to their Clair config.